### PR TITLE
add @media and change some code

### PR DESCRIPTION
**Pull request by SergiusHp**

### CODE REVIEW

File: style-media.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    .color-icon {
+        font-size: 22px;
+    }
+    .color_for_link {
+        font-size: 1.1rem;
+    }
+}
+
+@media (min-width: 1500px) {
+    .midle {
```
Comment (lapc002) [inline style-media.css]: Middle what ? it's a bad name just `middle` should be  - middle-block , middle-wrapper or something. Please rename

File: style.css [hunk 2]
```diff
@@ -214,12 +214,14 @@ body {
...
     text-decoration: none;
-    font-size: 0.8rem;
+    font-size: 1.5rem;
+    margin-left: 5px;
 }
 
 .color-icon {
     color: rgb(30, 119, 202);
     margin-left: 20px;
+    font-size: 30px;
```
Comment (lapc002) [inline style.css]: It's better to keep one strategy of sizing . You should use px or rem . 
Some font-size has px , another - rem . You need to keep something one.

File: style.css [hunk 3]
```diff
@@ -247,5 +249,4 @@ body {
     display: flex;
     flex-direction: row;
     justify-content: space-between;
-    width: 35%;
 }
```
Comment (lapc002) [inline style.css:252]: please add the new line in the end of file

File: style-media.css (hunk 1 above)
Comment (SergiusHp) [reply style-media.css]: I did

File: style.css (hunk 2 above)
Comment (SergiusHp) [reply style.css]: did

File: style.css (hunk 3 above)
Comment (SergiusHp) [reply style.css:252]: did

Comment (SergiusHp) [reply style.css:252]: but when i saved Visual studio dell new line

**Review by lapc002 — APPROVED** [review]

File: style.css (hunk 3 above)
Comment (lapc002) [reply style.css:252]: We need to fix it in the config

**Pull request merged**